chain-config/src/validator_rules.rs

```diff
@@ -17,18 +18,33 @@ pub struct TokenIdAmountPair<M: ManagedTypeApi> {
 #[multiversx_sc::module]
 pub trait ValidatorRulesModule: setup_phase::SetupPhaseModule + events::EventsModule {
     fn is_new_config_valid(&self, config: &SovereignConfig<Self::Api>) -> Option<&str> {
+        if let Some(additional_stake) = config.opt_additional_stake_required.clone() {
+            for stake in additional_stake {
+                require!(stake.token_id.is_valid_esdt_identifier(), INVALID_TOKEN_ID);
+            }
+        }
+
         if config.min_validators <= config.max_validators {
             None
         } else {
             Some(INVALID_MIN_MAX_VALIDATOR_NUMBERS)
         }
     }
 
+    #[payable]
     #[endpoint(register)]
     fn register(&self, new_validator: ValidatorInfo<Self::Api>) {
```

Comment:
```suggestion
    fn register(&self, bls_key: ManagedBuffer) {
```
You can create `ValidatorInfo` from caller + bls_key + payment